src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
yeah I think you'll only be able to swap attached views.

we can handle two parents in `swapEl`, but that'll be a bit weird for a single collectionview to have multiple parents.. but I guess if you've overridden attachHtml it is possible